### DISCUSSION TITLE
Dynamic render partial view

### DISCUSSION THREAD

**Discussion by sagarmumbaidev** [livewire/livewire#1379] [4. General]
Is there any way to render partial view in livewire? 

My condition is I want to render the multiple form with if else condition

**Comment by mblarsen** [comment]
You can use `@livewire($componentName, [... props ...])`

**Reply by sagarmumbaidev** [reply]
@mblarsen thanks for the reply, but what if I don't want to create a new livewire component and use the normal laravel view to render.
